pytorch_toolkit/ote/ote/metrics/detection/common.py

```diff
@@ -24,9 +34,8 @@ def collect_ap(path):
     return average_precisions
 
 
-def coco_ap_eval(config_path, work_dir, snapshot, update_config, show_dir=''):
+def coco_ap_eval(config_path, work_dir, snapshot, update_config, show_dir='', **kwargs):
```

Comment:
are these kwargs needed?

Reply:
Unfortunately some metric functions have a different set of parameters - for example `compute_wider_metrics` need at `wider_dir` parameter, mmdetection-based metrics uses `show_dir` parameter. The option to pass them all into all function contradicts the main idea of proposed refactoring - split general functions into SOLID classes.

Reply:
Ok, it should be redesigned, but it is ok for this PR